split-webapp/split/src/pages/Split.js

```diff
@@ -48,9 +47,24 @@ class Split extends Component {
         
         let bill = {
             title: "Transaction title",
+            payer: "",
+            total: cost,
             payments: paymentArray,
-            payed: ""
         }
+
+        this.createBill(bill);
+    }
+
+    createBill(data) {
+        return fetch('/api/bill_exec/create_bill', {
```

Comment:
Is return required here?

Reply:
Jake and I were not too sure on how to get data using the fetch method, so we took some inspiration from an example. Return in this case is not necessary, thank you for pointing it out!